docs/tactics.md

```diff
@@ -186,6 +186,23 @@ example {α : Type*} {a b : α} [add_comm_group α] (hyp : a + a - a = b - b) :
 by { abel at hyp, exact hyp }
 ```
 
+### norm_num
+
+Normalises numerical expressions. It supports the operations
+  `+` `-` `*` `/` `^` `<` `≤` over ordered fields (or other
+  appropriate classes), as well as `-` `/` `%` over `ℤ` and `ℕ`.
+
+In practice, this means that certain goals of the form `A = B` or `A ≠ B`, where `A` and `B` are
+numerical expressions, can be solved with `norm_num`. The tactic may succeed even if `A` and `B`
+are real numbers, and may not time out even if the numerals involved are large.
+```lean
+import data.real.basic
+
+example : (2 : ℝ) + 2 = 4 := by norm_num
+example : (12345.2 : ℝ) ≠ 12345.3 := by norm_num
+example : 123456789 + 987654321 = 1111111110 := by norm_num
```

Comment:
Examples with inequalities, prime, etc would be nice, as well as examples where the goal isn't proved but numeral sub-expressions are normalized.

Reply:
OK, I added a few more examples. I don't know what you mean about examples where the goal isn't proved but numerical sub-expressions are normalised -- is `norm_num` safe to use in the middle of a proof? I only ever use it to close goals. Do you mean this:
`example (x : ℝ) (h : x = 123 + 456) : x = 579 := by norm_num at h; assumption`?